modules/platforms/dotnet/Apache.Ignite/Internal/Linq/ResultSelector.cs

```diff
@@ -75,6 +77,16 @@ public static RowReader<T> Get<T>(IReadOnlyList<IColumnMetadata> columns, Expres
                 static k => EmitConstructorReader<T>(k.Target, k.Columns, k.Options));
         }
 
+        if (selectorExpression is MemberInitExpression memberInitExpression)
+        {
+            var ctorInfo = memberInitExpression.NewExpression.Constructor!;
+            var cacheTarget = new MemberInitCacheTarget(ctorInfo, memberInitExpression.Bindings.Select(b => b.Member).ToList());
```

Comment:
Let's pass `memberInitExpression.Bindings` to the `MemberInitCacheTarget` constructor to avoid list allocation.

Reply:
Done.